CONTRIBUTING.md

```diff
@@ -325,6 +328,113 @@ if you change the history (rebase, squash, amend), and use git push --force to
 update the branch on your fork. The pull request points to that branch, not to
 specific commits.
 
+## Exercism Web Structure
+
+### Intro
+
+Exercism is built with the [Sinatra](https://github.com/sinatra/sinatra) web framework.
+If you haven't heard of it, definitely check it out. It's an awesome lightweight web framework for Ruby.
```

Comment:
I agree that it's awesome, but let's simplify to just "It's a lightweight web framework in Ruby".